src/components/TicTacToe.tsx

```
@@ -0,0 +1,98 @@
+      <div className={styles.board}>
+        {Array(3)
+          .fill(null)
+          .map((_, rowIndex) => (
+            <div key={rowIndex} className={styles.boardRow}>
+              {Array(3)
+                .fill(null)
+                .map((_, colIndex) => renderSquare(rowIndex * 3 + colIndex))}
+            </div>
+          ))}
+      </div>
+      <button className={styles.resetButton} onClick={resetGame}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)